src/main/java/com/google/cloud/pontem/Util.java

```diff
@@ -153,10 +155,24 @@
 
   /**
    * Return structs from a single query. Result structs must fit into memory, so these cannot be
-   * huge queries.
+   * huge queries. If you want to just use the current timestamp, pass Timestamp.now()
    */
-  public ImmutableList<Struct> performSingleSpannerQuery(
+  public ImmutableList<Struct> performSingleSpannerReadQuery(
       String projectId, String instance, String databaseId, String querySql) {
+    return performSingleSpannerReadQueryAtTimestamp(
+        projectId, instance, databaseId, querySql, Timestamp.now());
+  }
+
+  /**
+   * Return structs from a single query. Result structs must fit into memory, so these cannot be
+   * huge queries. If you want to just use the current timestamp, pass Timestamp.now()
```

Comment:
Maybe it's worth adding a comment detailing the benefits of specifying the read timestamps and what it's valid values are. I assume past values would fail but I'm not sure if that's the case.